metrics/network/remote_network/README.md

```diff
@@ -12,6 +12,9 @@ the maximum network bandwidth with `iperf3`.
 
 - The `remote-networking-qperf.sh` script measures TCP and UDP latency with the `qperf` tool.
 
+- The `direct-remote-networking-iperf3.sh` script measures bandwidth using a remote setup without using Swarm.
+Host A will run an `iperf3` server container while host B will run an `iperf3` client container.
```

Comment:
Suggested rewrite to make the sentence active: Host A runs an `iperf3` server container while host B runs an `iperf3` client container.